app/app/local_settings.py.dist

```diff
@@ -135,4 +135,11 @@ AWS_SECRET_ACCESS_KEY = 'TODO'
 S3_REPORT_BUCKET = 'TODO'
 S3_REPORT_PREFIX = 'TODO'
 
+# WEB3
+WEB3_HTTP_PROVIDER = 'TODO'
+
+# COLO Coin
+COLO_ACCOUNT_ADDRESS = 'TODO'
```

Comment:
@mbeacom could you PR a related change to the server repo with a new account for this?  from there i can fund it with some tokens / eth